### PR TITLE
New blog post: Technology Carbon Standard Update - 15th March 2024

### PR DESCRIPTION
**Pull request by kdavis04**
https://kdavis04.github.io/blog/2024/03/15/technology-carbon-standard-update.html

Have you (please tick each box to show completion):
- [x] Added your blog post to a single category?
- [x] Added a brief summary for your post? Summaries should be roughly two sentences in length and give potential readers a good idea of the contents of your post.
- [x] Checked that the build passes?
- [x] Checked your spelling (you can use npm spellcheck if that's your thing)
- [x] Ensured that your author profile contains a profile image, and a brief description of yourself? (make it more interesting than just your job title!)
- [x] Optimised any images in your post? They should be less than 100KBytes as a general guide.

Posts are approved based on their category. replace (at) with @ to notify them!

Tech - Colin Eberhardt - @ColinEberhardt

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -546,6 +547,10 @@ authors:
     author-summary: "I am a Test Engineer from Newcastle"
     name: "Kay Coombes"
```
Comment (dhope-scottlogic) [inline _data/authors.yml:550]: Does this need email and feed-description? Not sure if mandatory according to the confluence page, just that I have them in mine

Comment (kdavis04) [reply _data/authors.yml:550]: I based this on a lot of the other authors so I'm assuming those aren't mandatory - happy to add if required

Comment (kdavis04) [reply _data/authors.yml:550]: Added these fields now 👍

Comment (dhope-scottlogic) [reply _data/authors.yml:550]: Ok, visually the changes all look good to me now

**Pull request merged**